.github/workflows/rust-wasmedge-macro-release.yml

```diff
@@ -5,62 +5,83 @@ concurrency:
   cancel-in-progress: true
 
 on:
+  # workflow_dispatch:
```

Comment:
Never comments out something in the workflow. If you think this is helpless, then remove it.